README.md

```diff
@@ -7,6 +7,17 @@ A python implementation of the UA Parser (https://github.com/ua-parser, formerly
 
 ##Installing
 
+### Install via pip
+First, check that you've got the correct dependencies. For example, on Ubuntu:
+```
+$ apt-get install libyaml libyaml-dev python-dev
```

Comment:
This is completely untrue. I just verified with our latest release that it works just fine without `libyaml` or `python-dev`.

Can you provide me with some steps to reproduce?

This was with:

* pip 7.1.2
* python 2.7.9
* ua-parser 0.5.1

on Debian Jessie.